### DISCUSSION TITLE
Grafana Dashboard for ClickHouse Backup Representation

### DISCUSSION THREAD

**Discussion by karthik-thiyagarajan** [Altinity/clickhouse-backup#975] [Q&A] [answered]
Is there a grafana dashboard available to represent Clickhouse Backup execution / failure ? Is this data being exported by the operator metrics? Can you help to identify the right metrics ?

**Comment by Slach** [comment, marked as answer]
> Is there a grafana dashboard available to represent Clickhouse Backup execution / failure ? Is

No dashboard now, don't be shy to make Pull Request

> Is this data being exported by the operator metrics

no, clickhouse-backup have separate :7171/metrics endpoint

> Can you help to identify the right metrics ?

look prometheus alerts in https://github.com/Altinity/clickhouse-operator/blob/master/deploy/prometheus/prometheus-alert-rules-backup.yaml
